exchanges/huobi/huobi.go

```diff
@@ -20,9 +20,12 @@ import (
 
 const (
 	huobiAPIURL      = "https://api.huobi.pro"
+	huobiURL         = "https://api.hbdm.com/"
 	huobiAPIVersion  = "1"
 	huobiAPIVersion2 = "2"
 
+	huobiSwapMarkets           = "/swap-api/v1/swap_contract_info"
```

Comment:
This will have an extra slash in it when combined with `huobiURL`.  This will destroy the very foundation of our existence and is not a petty nit at all 👀